apps/docs/project.json

```diff
@@ -34,6 +34,22 @@
             "891377307993.dkr.ecr.eu-west-1.amazonaws.com/life-events-design-system:prd",
             "891377307993.dkr.ecr.eu-west-1.amazonaws.com/life-events-design-system:latest"
           ]
+        },
+        "uat": {
```

Comment:
can define `staging` before `uat`, so it matches order of deployment (staging deployments before uat)